src/index.ts

```diff
@@ -413,49 +396,47 @@ export class GalleryComponent extends BaseComponent {
       thumb.initialHeight = medianHeight;
     }
 
-    this._$thumbs.link($.templates.galleryThumbsTemplate, this._thumbs);
+    const renderedHtml = this._thumbs.map(this._galleryThumbsTemplate).join('');
+    this._$thumbs.html(renderedHtml);
 
     if (multiSelectState && !multiSelectState.isEnabled) {
       // add a selection click event to all thumbs
       this._$thumbs.delegate(".thumb", "click", function (e: any) {
         e.preventDefault();
-        const thumb = $.view(this).data;
-        that.fire(Events.THUMB_SELECTED, thumb);
+        const thumbIndex = parseInt($(this).attr("data-index") as string);

```

Comment:
No longer accessing views, so manually get the data index

Reply:
Do we need to use jQuery here, or could we potentially do this instead?

```suggestion
        const thumbIndex = parseInt(this.dataset.index as string);
```

Maybe I'm misunderstanding what `this` is in this context, in which case feel free to disregard my suggestion. Seems like the native JS way might be more efficient if it's workable, though!